app/webpack/observations/identify/components/observation_modal.jsx

```diff
@@ -671,23 +671,33 @@ class ObservationModal extends React.Component {
             <ul className="inat-tabs">
               {activeTabs.map( tabName => (
                 <li key={`obs-modal-tabs-${tabName}`} className={activeTab === tabName ? "active" : ""}>
-                  <button
-                    type="button"
-                    className="btn btn-nostyle"
-                    onClick={e => {
-                      e.preventDefault( );
-                      chooseTab( tabName, { observation } );
-                      return false;
-                    }}
-                  >
-                    {
+                  {activeTabs.length > 1 ? (
+                    <button
+                      type="button"
+                      className="btn btn-nostyle"
+                      onClick={e => {
+                        e.preventDefault( );
+                        chooseTab( tabName, { observation } );
+                        return false;
+                      }}
+                    >
+                      {
                       tabTitles[tabName]
                       || translateWithConsistentCase(
                         _.snakeCase( tabName ),
                         { case: "upper", defaultValue: tabName }
                       )
                     }
-                  </button>
+                    </button>
+                  ) : (
+                    <span className="btn-nostyle active">
+                      { tabTitles[tabName]
+                      || translateWithConsistentCase(
+                        _.snakeCase( tabName ),
+                        { case: "upper", defaultValue: tabName }
+                      ) }
```

Comment:
The content of the tag is identical even through the elements wrapping it changes, so pull that out into a variable so you're only setting it once.